src/com/article-changer-v3.js

```diff
@@ -121,73 +122,74 @@
 					
 					if (!nextPage.length) {
 						App.log({
-							args: 'Could not find new article',
+							args: 'Could not find new article with this handle: ' + (!!newPageHandle ? newPageHandle : '(empty handle)'), // jshint ignore:line
```

Comment:
`args` can be an array. so:

```suggestion
							args: [
								'Could not find new article with this handle: ',
								(!!newPageHandle ? newPageHandle : '(empty handle)')
							],
```